pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
> I can't find anything like NamedDeleter (only in upstream comment). Does it mean we have to create it upstream?

hehe.  @smarterclayton named subresources? :)

Lacking a named subresource (I doubt @smarterclayton will allow that, but it would be nice), you'll be promoting this to a top level entity if we think that deletion is essential.

You need the name to be 1-to-1 mappable to the signature, probably in the forward direction.  Again, without named subresource, things get weird.  You'll need to include and parse out image.Name, signature.Type, signature.Content into a string we can parse out of a URL.

We could also consider adding a constrained `Name` field to the signature that is settable by callers.  That probably makes more sense.  Then you'll need `image.Name` and `signature.Name` to make a working top-level resource.